### ISSUE TITLE
Food52 failed scrape "site failed to return valid schema"

### ISSUE THREAD

**Issue by sckilcoyne** [mealie-recipes/mealie#779]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

Attempted to scrape https://food52.com/recipes/38745-vermont-spice-pumpkin-cake but it failed. On Test Scrape I get 
`{
  "details": "site failed to return valid schema"
}`

Looking at the source of the page, I do see that it should be scrape-able based on the error message Mealie gives
 `<script type="application/ld+json">
  {
    "@context": "http://schema.org/",
    "@type": "Recipe",
    "mainEntityOfPage": "https://food52.com/recipes/38745-vermont-spice-pumpkin-cake",
    "name": "Vermont Spice Pumpkin Cake",`

### Deployment

Docker (Synology)

### Mealie Version

0.5.3

**Comment by cadamswaite** [comment]
Yeah that website should be scrapeable, it works (both importing and the test scrape) on my 0.5.3 and on [the beta site](https://beta.mealie.io/recipe/vermont-spice-pumpkin-cake). I can't currently get onto the demo site to test there, but they should all be on the same scraper.
There is an issue with scraping the image though - I'll try to debug that tomorrow.

In the meantime, please could you provide some logs to help find where the error is coming from?

**Comment by sckilcoyne** [comment]
Sure, where do I find the logs?

**Comment by cadamswaite** [comment]
It depends a little on your setup, the easiest way would usually be through the filesystem. E.g. if you have something like this in your docker setup
```
volumes:
      - ./data/:/app/data
  ```
Then you should see a file called 'mealie.log' in the data folder.
You can also do it through docker command line if you're that way inclined using `docker ps` and `docker logs`. 
You don't have to send the whole file, just the bits from around the time of the error would be useful

Have you had any issues importing from other sites?

**Comment by sckilcoyne** [comment]
I figured out what my problem was. When I went to the logs, there was no evidence of attempting to create a new recipe, though the logs were being updated for things like restarting the docker container. I then tried another recipe from a site I had previously used and that failed the same way (same error message with no logs). The problem was I changed my firewall settings and that was blocking the service. Disabling the firewall allowed me to scrape recipes again.

Maybe the bug should be logs not capturing failed recipe attempts?
